tembo-stacks/src/stacks/specs/analytics.yaml

```diff
@@ -1,12 +1,11 @@
 name: Analytics
-description: A Postgres instance equipped ParadeDB's pg_analytics extension.
+description: A Postgres instance optimized for analytics workloads.
 repository: "quay.io/tembo"
 organization: tembo
 images:
-  14: "standard-cnpg:14-30219f2"
-  15: "standard-cnpg:15-30219f2"
-  16: "standard-cnpg:16-30219f2"
-  17: "standard-cnpg:17-30219f2"
+  14: "dw-cnpg:14-30219f2"
+  15: "dw-cnpg:15-30219f2"
+  16: "dw-cnpg:16-30219f2"
```

Comment:
Not ready for 17 yet?

Reply:
I think columnar doesn't support PG17 yet.